.github/workflows/CompatHelper.yml

```
@@ -3,6 +3,11 @@ on:
   schedule:
     - cron: 0 0 * * *
   workflow_dispatch:
+  workflow_call:
+    secrets:
+      DOCUMENTER_KEY:
+        description: 'An SSH deploy key. See docs for more info.'
+        required: true
```

Is this actually required? I'm never sure.

If not:

```suggestion
        required: false
```